### DISCUSSION TITLE
Fritzing für ESP32-PoE

### DISCUSSION THREAD

**Discussion by wib100** [tbnobody/OpenDTU#297] [General]
Hallo,
ich habe ein Fritzing für den Olimex ESP32-PoE(-ISO) erstellt und würde es gerne allen zur Verfügung stellen.
![OpenDTU-ESP32-PoE-Wiring](https://user-images.githubusercontent.com/35136309/199605649-12ece42d-df1d-4229-9a7a-020d15acec51.png)

**Comment by dersch81** [comment]
Bevor es verloren geht würde ich ein PR für die Readme hier erstellen.

**Comment by D3R-ST3FAN** [comment]
Ich dachte die GPIOs wurden bei der Variante für den Olimex ESP32-POE (olimex_esp32_poe) extra angepasst um den UEXT Connector zu nutzen.
So hab ich das jedenfalls gemacht. Willst Du das gleich mit aufnehmen?

siehe https://www.olimex.com/Products/IoT/ESP32/ESP32-POE/resources/ESP32-POE-GPIO.png

**Comment by vo5tr0** [comment]
Hat schon jemand den UEXT Port zum laufen bekommen? Bei mir klappt es leider nicht :(
 Ich bin mir unsicher, aber ich denke es hängt am GPI36 Pin (kein GPIO)

**Comment by D3R-ST3FAN** [comment]
Also beim  Olimex ESP32-POE Build sind die Pins doch entsprechend gemapped (siehe Pinout)

CE=16        IRQ=13
MISO=15   MOSI=2
SCLK=14    PIN_CS=5

So hab ich das Modul auch angeschlossen am UEXT.
GPI36 und GPI4 sind der UART1 und werden gar nicht benutzt. und bleiben frei.

**Reply by vo5tr0** [reply]
Dann teste ich das nochmal. Schade, dass die beiden Pins so mitten drin sind. Sonst könnte man die Antenne direkt auf der anderen Seite von dem Kabel einstecken. 

Hast du es dann mit diesen Jumper Kabeln gelöst oder gibt es da eine schönere Möglichkeit? :)

**Comment by tmonxx** [comment]
Moin moin.

Eine aktuelle Frage zu einem eingeschlafenen Thread. Ist das möglich diese ESP32 POE Lösung auch im Dualbetrieb zu nutzen? D.h. CMT und NRF Module gleichzeitig.

**Comment by cored0wn** [comment]
Entschuldigt wenn das eine dumme Frage ist, aber wo kann ich die Datei herunterladen? Ich sehe nur dein Bild.

**Reply by martin-snajdr** [reply]
Das wüsste ich auch gerne.
